rest/resource-server/src/docs/asciidoc/licenses.adoc

```diff
@@ -96,6 +96,20 @@ include::{snippets}/should_document_update_license/http-response.adoc[]
 ===== Links
 include::{snippets}/should_document_update_license/links.adoc[]
 
+[[resources-update-whitelist-license]]
+==== Update whitelist a license
```

Comment:
can you change the heading to, Update whitelist for license's obligations for better understanding.

Reply:
@rudra-superrr  Thanks for your comment. I have fixed it. Please help me to review the changes